source/developer-guides/build-system.rst

```diff
@@ -90,15 +104,170 @@ Slim Bootloader is built in stages and more information on each stage is given b
   * **Packaged As**: FD containing Stage2 FV, and FSP-S binary as a FILE
   * **Stage 2 FV**: Contains ACPI Table, Vbt, Logo, and Stage2 PEIM
 
+* **OsLoader**:
+
+  * **Packaged As**: FD containing OsLoader FV
+
+* **FwUpdate**:
+
+  * **Packaged As**: FD containing Firmware Update FV
+  * Note that FwUpdate is included only if ``ENABLE_FWU`` is enabled in ``BoardConfig.py``
+
+* In addition to the Slim Bootloader Stages and the payloads, the final SBL image may include some other components like Microcode binaries,
+  ACM binary, SBL container binaries, etc.
 
 .. _post-build:
 
+Post Build Image Construction
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Each stage and component of Slim Bootloader is individually built and then put together into one image at the end
+  of the build process.
+* The final image layout for Slim Bootloader can be defined in ``BoardConfig.py`` - ``GetImageLayout()``
+
+  * If firmware resiliency is enabled, the layout will typically contain redundant copies of the boot-critical code.
+    Check :ref:`firmware-resiliency-and-recovery` for more details.
+
+* As discussed above, the final SBL image includes SBL stages and other components, examples of which are listed below:
+
+  * These additional components included in the SBL image are placed in the non-redundant region of the image
+  * The list of included containers or components in the final image is present in the ``GetImageLayout()`` function
+  * Examples:
+
+    * ``ACM.bin`` - Authenticated Code Module
+    * ``MRCDATA.bin`` - Memory Reference Code Data, used to store Memory Reference Code training data
+    * ``VARIABLE.bin`` - This region is used for ``GetVariable()`` and ``SetVariable()`` APIs provided by ``LiteVariableLib`` in ``BootloaderCommonPkg``
+    * ``UCODE.bin`` - CPU Microcode
+
+    * The components listed above are generated by the ``post_build()`` function in ``BuildLoader.py``
+
+  * Payloads other than OsLoader and FW Update are built independently and the payload binary is packaged as ``EPAYLOAD.bin``
+    to be included in the final SBL image
+
+    * ``EPAYLOAD.bin`` - Payload for SBL - used for extra payloads (UEFI Payload, Linux, u-boot, etc.)

```

Comment:
Please add a reference to - https://slimbootloader.github.io/how-tos/integrate-multiple-payloads.html#integrate-multiple-payloads

Reply:
Added, thanks